dev/pr-auditor/main.go

```diff
@@ -167,6 +181,8 @@ func preMergeAudit(ctx context.Context, ghc *github.Client, payload *EventPayloa
 		prState = "failure"
 		stateDescription = "No test plan detected - please provide one!"
 		stateURL = "https://docs.sourcegraph.com/dev/background-information/testing_principles#test-plans"
+	case result.ProtectedBranch:
+
```

Comment:
```suggestion
	case result.ProtectedBranch:
		prState = "success"
		stateDescription = "No action needed, but an exception will be opened post-merge."
```

You can also set a `stateDescription` and `prState`